DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -23,39 +23,39 @@ final class SRSnapshotTests: SnapshotTestCase {
     }
 
     func testBasicTexts() throws {
-        try takeSnapshotFor(.basicTexts, with: [.allow, .mask, .maskUserInput], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(.basicTexts, with: allTextAndInputPrivacyLevels, shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
     }
 
     func testSliders() throws {
-        try takeSnapshotFor(.sliders, with: [.allow, .mask, .maskUserInput], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(.sliders, with: allTextAndInputPrivacyLevels, shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
     }
 
     func testProgressViews() throws {
-        try takeSnapshotFor(.progressViews, with: [.allow, .mask], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(.progressViews, with: [.maskSensitiveInputs, .maskAll], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
     }
 
     func testActivityIndicators() throws {
-        try takeSnapshotFor(.activityIndicators, with: [.allow, .mask], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(.activityIndicators, shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
```

Comment:
I don't think we need to test different privacy levels here, which is why it's only testing the default one.